Tests/IntegrationTests/IntegrationTests.csproj

```diff
@@ -141,7 +141,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="App.config" />
+    <None Include="..\..\RoslynPluginGenerator\App.config">
+      <Link>App.config</Link>
+    </None>
```

Comment:
Changed duplicated files inside multiple projects to links back to the "true" product file, avoiding the possibility of getting out of sync.